backend/endpoints/datasets/bundle.sh

```diff
@@ -0,0 +1,9 @@
+pip install --requirement=datasets/requirements.txt --target=/asset-output \
```

Comment:
If you `set -o errexit` at the top of the file it'll achieve the same as chaining the commands with `&&` - it'll stop executing as soon as one of the commands returns a non-zero exit code.

Reply:
Fixed in 01730f4e95b2905ac868c2fb1b09f515c10e9871